client/src/pages/Sights.js

```diff
@@ -1,12 +1,37 @@
+import { useState, useEffect } from 'react';
 import styled from 'styled-components/macro';
 
+import Filter from '../components/Filter';
 import SightCard from '../components/SightCard';
 
 export default function Sights({ sights, toggleFavorite, isFave }) {
+  const counties = [
+    { label: 'Antrim (UK)', value: 'antrim (uk)' },
+    { label: 'Clare', value: 'clare' },
+    { label: 'Cork', value: 'cork' },
+    { label: 'Dublin', value: 'dublin' },
+  ];
+  const [selected, setSelected] = useState(counties);
+  const [selectedSights, setSelectedSights] = useState(sights);
+
+  useEffect(() => {
+    const filteredSights = sights.filter((sight) =>
+      selected.find((county) => county.value === sight.county.toLowerCase())
+    );
+    setSelectedSights(filteredSights);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Is this necessary?

Reply:
it was necessary when I wrote the useEffect, because otherwise eslint would show me a warning, but now it works without it somehow :D